### PR TITLE
Add POST method , try with postman and have an error #3

### PR DESCRIPTION
**Pull request by ChauPham90**

### CODE REVIEW

File: e-commerce/server/models/user.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    name: {
+        type: String,
+        strim: true,
+        required:true
+
+    },
+    email: {
+        type: String,
+        strim: true,
+
```
Comment (MayaPancake) [inline e-commerce/server/models/user.js]: whats missing here?

Comment (MayaPancake) [reply e-commerce/server/models/user.js]: you need this: required: true

**Review by MayaPancake — CHANGES_REQUESTED** [review]
I commented on the code that you are missing.

File: e-commerce/server/models/user.js (hunk 1 above)
Comment (ChauPham90) [reply e-commerce/server/models/user.js]: did u solve it?

**Pull request merged**